src/server/database/migrations/20180423095250_create-projects.js

```diff
@@ -0,0 +1,23 @@
+export const up = async knex => {
+  await knex.schema.createTable('projects', table => {
+    table
+      .increments('id')
+      .unsigned()
+      .primary()
+    table.string('name', 30).notNull()
+    table
+      .string('slug')
+      .unique()
+      .notNull()
+    table.text('description')

```

Comment:
Should this simply be a `string('description')`? 255 characters is more than enough.